### ISSUE TITLE
Error loading manage customers

### ISSUE THREAD

**Issue by guztavoalves** [sitewards/B2BProfessional#43]
After installing the extension, when I try to access admin> Customer> customers manage, the list does not charge.

When access:
System> Configuration> Client configuration, the page does not load, opening a blank page.

When you uninstall the extension, everything returns to normal.

What may be occurring?

**Comment by dmanners** [comment]
Hi @guztavoalves sorry to hear you are having this problem.

Can you let me know which controllers you are trying to access? Full path for the controller would really help investigating this issue.

**Comment by guztavoalves** [comment]
Yes, but for some reason the control back pages to be loaded, the paths are:

* Magento Admin > System> Configuration> Customers> Customer Configuration
* Magento Admin > Customers> Manage Customers

I am also unable to enable the registration of clients to use with the extension in:

* Magento Admin > Customers> Manage Customers

By clicking Actions to change the status, loads a blank page and nothing else is loaded.

I'm using Magento 1.9

-x-x-x-x-

I tried to use this extension in a VISION of my shop and the asset it, products within the pages of categories ALL VISIONS not load more, even I enabling the extension for only one view.

**Comment by papillon6** [comment]
Any news or workarounds on this topic?

**Comment by papillon6** [comment]
Found the error:
In my case a php-file was missing:

./app/code/community/Netzarbeiter/CustomerActivation/Block/Adminhtml/Widget/Grid/Column/Renderer/Boolean.php

Cloning the repository on my webserver leaded to:

/app/code/community/Netzarbeiter/CustomerActivation/Block/Adminhtml/Widget/Grid/Column/Renderer/Boolean.php/Boolean.php

Which means the first "boolean.php" was a directory and the second "boolean.php" was the original file inside this directory.  I fixed it manually. Errors in backend are now solved.

If I clone the repository on my windows machine everything goes fine...?

**Comment by MarceloAntunes** [comment]
In the installation did not create the file "boolean.php", created only the folder Boolean.php.
Where can I get this file? Do you know the code that's inside it?

**Comment by andrewhowdencom** [comment]
Hi @MarceloAntunes; 

That file was removed in this commit: https://github.com/sitewards/B2BProfessional/commit/fcff79451339e0c2925144580594413e940291bd

It should be provided by the upstream package, specified in composer.json: 

https://github.com/Vinai/customer-activation/blob/master/app/code/community/Netzarbeiter/CustomerActivation/Block/Adminhtml/Widget/Grid/Column/Renderer/Boolean.php

Can you confirm this file is not being created as expected when just the extension is installed?

Cheers!
